x/oracle/genesis.go

```diff
@@ -0,0 +1,20 @@
+package oracle
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/umee-network/umee/x/oracle/keeper"
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// InitGenesis initializes the x/oracle module's state from a provided genesis
+// state.
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
+	k.SetParams(ctx, genState.Params)
```

Comment:
Is there nothing else to do when initializing genesis?

Reply:
There's more to do here & `ExportGenesis`, just wanted to add it after I started pruning the stuff we don't need - I can open an issue for it